ui/guest/src/react/Guest.tsx

```diff
@@ -486,13 +485,13 @@ function Guest(props: GuestProps) {
               <FieldInstanceSwitcher
                 onNext={() =>
                   dispatch({
-                    type: CONTENT_TREE_SWITCH_FIELD_INSTANCE,
+                    type: contentTreeSwitchFieldInstance.type,
                     payload: { type: 'next', scrollElement }
                   })
                 }
                 onPrev={() =>
                   dispatch({
-                    type: CONTENT_TREE_SWITCH_FIELD_INSTANCE,
+                    type: contentTreeSwitchFieldInstance.type,
                     payload: { type: 'prev', scrollElement }
```

Comment:
There are too many instances — like this one — of action creators being used like a constant; here and in the epics. Probably the creators aren't typed either.